scripts/template_openxr_dispatchdynamic.hpp

```diff
@@ -59,30 +56,52 @@ class DispatchLoaderDynamic {
      * @name Constuctor/Factory functions
      * @{
      */
+/*{protect_proto_begin()}*/
```

Comment:
I think this doesn't need to be templated, it can just be `#ifndef XR_NO_PROTOTYPES` or whatever the macro is :)